doc/content/getting-started/migrating/migrating-from-v2/_index.md

```diff
@@ -0,0 +1,90 @@
+
+{{< note >}} OTAA end devices connected to The Things Network V2 can be migrated to {{% tts %}} Cloud and {{% tts %}} Community Edition without migrating a gateway, thanks to their default connection to [Packet Broker]({{< ref "/reference/packet-broker" >}}). The Things Industries V2 (SaaS) can be connected to Packet Broker by contacting [support](mailto:support@thethingsindustries.com), while {{% tts %}} Enterprise or Open Source can even be [configured]({{< ref "/reference/packet-broker/connect" >}}) by a user.
+
+This means that if you are using deployments connected to Packet Broker, the traffic from your OTAA end device will be immediately available in {{% tts %}} after you successfully migrate your end device.
+
+On the other hand, if you are using deployments that are not connected to Packet Broker, you will have to [migrate your gateway]({{< ref "/getting-started/migrating/gateway-migration" >}}) as well in order to receive traffic from your OTAA end device on {{% tts %}}. You will need to migrate your gateway if you are migrating an ABP device as well. {{</ note >}}
+
+## Prerequisites
+
+1. User account in {{% ttnv2 %}}.
+2. End device(s) and gateway(s) connected to {{% ttnv2 %}} network.
+3. User account in {{% tts %}}.
+
+{{< note >}} We **highly recommend** using {{% tts %}} version `3.12.0` or higher. Some of the features (like session migration) from this guide might not be available for prior versions. {{</ note >}}
+
+## Packet Broker Role in Migration Process
+
+Before we start with explaining the process of migration, it is important to understand the role of the [Packet Broker]({{< ref "/reference/packet-broker" >}}) in it.
+
+End devices connected to **The Things Network V2** can be migrated to **{{% tts %}} Cloud** and **{{% tts %}} Community Edition** without migrating a gateway, thanks to their default connection to Packet Broker. **The Things Industries V2 (SaaS)** deployments can also be connected to Packet Broker on customer request by contacting [our support](mailto:support@thethingsindustries.com), while **{{% tts %}} Enterprise** or **{{% tts %}} Open Source** can even be [configured]({{< ref "/reference/packet-broker/connect" >}}) by a user to connect to Packet Broker.
+
+If you are using deployments connected to Packet Broker and your gateway is still connected to {{% ttnv2 %}}, the traffic from your end device can be routed to {{% tts %}} (and reverse) after you successfully migrate only your end device. In order for this traffic to be properly routed by the Packet Broker, there are some requirements your devices need to fulfill:
+
+- {{% ttnv2 %}} **DevAddr** of your end device needs to be routable by the Packet Broker.
+    
+    OTAA devices migrated without an active session will acquire a new **DevAddr** from {{% tts %}} during the join procedure. This **DevAddr** will always be routable by the Packet Broker.
```

Comment:
I see indentation but this isn't a list

Reply:
Also I don't understand this sentence. What does this mean for the user?

Reply:
I imagined this text under the line in 36 (that has a bullet) to be the explanation of the sentence in line 36. Since I'm talking about requirements for Packet Broker, I don't see these bullets as subsections, but it looks kind of ugly without indentation. 

With indentation:
![Screen Shot 2021-04-21 at 16 02 21](https://user-images.githubusercontent.com/65215579/115409207-d5bc2680-a1f1-11eb-86fe-c1558c87151e.png)

Without indentation:
![Screen Shot 2021-04-21 at 16 02 21](https://user-images.githubusercontent.com/65215579/115409244-dc4a9e00-a1f1-11eb-9bee-1d06ada0e33b.png)

I can ofc remove the indentation, or separate this part from other text in some other way

Reply:
> Also I don't understand this sentence. What does this mean for the user?

That sentence should mean that if the user has an OTAA device, migrates it without an active session (i.e. makes it perform a new join on V3), the device will get a new DevAddr which won't be a problem for Packet Broker to route, so the user doesn't have to worry about it.

If I understood correctly from all the input I got, if the device is migrated with its active session, the DevAddr (from V2) can be from an address block that's basically unroutable by the Packet Broker.